exchange-contract/docs/notebooks/factories/wallet.py

```diff
@@ -0,0 +1,59 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.16.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# + [markdown] editable=true slideshow={"slide_type": ""}
+# # Wallet Factory
+#
+# Use this notebook to create a wallet associated with a particular issuer.
+
```

Comment:
I am  afraid I do not understand which contract this notebook is going to interact with ? Is it a placeholder?

Reply:
placeholder. again, i need to label a bunch of these as WIP.

Reply:
Marked as WIP